spec.html

```diff
@@ -324,7 +322,7 @@
       <h1>BigInt::signedRightShift (_x_, _y_)</h1>
       <p>The abstract operation BigInt::signedRightShift with arguments _x_ and _y_ of type BigInt:</p>
       <emu-alg>
-        1. Return ? BigInt::leftShift(_x_, -_y_).
+        1. Return BigInt::leftShift(_x_, -_y_).
```

Comment:
Never mind, really there's no way that this could possibly throw. I'll take your change here.